crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -1175,15 +1193,52 @@ mod tests {
         (
             temp_dir,
             CompressedDataManager::try_new(
-                None,
+                RwLock::new(None),
                 local_data_folder,
+                channels,
                 memory_pool,
-                Arc::new(RwLock::new(Metric::new())),
+                Arc::new(Mutex::new(Metric::new())),
             )
             .unwrap(),
         )
     }
 
+    /// Return a [`CompressedSegmentBatch`] containing three compressed segments.
+    fn compressed_segments_record_batch() -> CompressedSegmentBatch {
+        compressed_segment_batch_with_time(0, 0.0)
+    }
+
+    /// Return a [`CompressedSegmentBatch`] containing three compressed segments from univariate_id.
```

Comment:
Minor but maybe remove `from univariate_id` since it makes it sound like `univariate_id` is an argument and not just `COLUMN_INDEX`.

Reply:
Fixed.